internal/seclang/directives.go

```diff
@@ -876,6 +876,8 @@ func directiveSecUploadDir(options *DirectiveOptions) error {
 	return nil
 }
 
+// Not implemented yet
```

Comment:
Not sure this will work for autogeneration. Everything not being key:value (e.g. Description: Configures....) should go after the
```
// ---
```

Reply:
Thanks, moved as `// Note: not implemented yet` as the last line